### PR TITLE
Adding a sample for FooterMenuItems

### PR DESCRIPTION
**Pull request by anawishnoff**
## Description

Added a sample on the NavigationView page to show off the new FooterMenuItems feature, coming in WinUI 2.5. More info on this feature can be found in [the spec](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/active/NavigationView/BottomItems/BottomMenuItems.md).

## Motivation and Context

As a part of releasing this feature, it should have a corresponding sample. 

## How Has This Been Tested?

Just tested manually by me, using mouse, touch, and keyboard in light and dark modes.

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/ControlPages/NavigationViewPage.xaml [hunk 1]
```diff
@@ -224,6 +224,53 @@
...
+        <local:ControlExample HeaderText="NavigationView with Footer Menu Items"
+                              VerticalAlignment="Top" 
+                              HorizontalContentAlignment="Stretch"
+                              XamlSource="NavigationView\NavigationViewSample9_xaml.txt"
+                              WebViewHeight="950">
+            <local:ControlExample.Options>
+                <StackPanel>
+                    <TextBlock Text="PanePosition:" Margin="0,12,0,0"/>
+                    <RadioButton x:Name="nvSample9Left" GroupName="FooterGroup" Content="Left mode" Checked="panePositionLeft_Checked" IsChecked="True"/>
+                    <RadioButton x:Name="nvSample9Top" GroupName="FooterGroup" Content="Top mode" Checked="panePositionTop_Checked"/>
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/NavigationViewPage.xaml:236]: Mostly thinking out loud: should this use RadioButtons control instead?

File: XamlControlsGallery/ControlPages/NavigationViewPage.xaml.cs [hunk 2]
```diff
@@ -197,6 +198,22 @@ private void NavigationView_SelectionChanged8(Microsoft.UI.Xaml.Controls.Navigat
             }
         }
 
+        private void NavigationView_SelectionChanged9(Microsoft.UI.Xaml.Controls.NavigationView sender, Microsoft.UI.Xaml.Controls.NavigationViewSelectionChangedEventArgs args)
+        {
+            if (args.IsSettingsSelected)
+            {
+                contentFrame9.Navigate(typeof(SampleSettingsPage));
+            }
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPages/NavigationViewPage.xaml.cs:206]: The Settings item is hidden, so you actually don't need this if clause.

File: XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample9_xaml.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<muxc:NavigationView x:Name="nvSample9" 
+                            Grid.Row="1"
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample9_xaml.txt:2]: Remove this line, it's not adding any value to the sample code

File: XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample9_xaml.txt [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<muxc:NavigationView x:Name="nvSample9" 
+                            Grid.Row="1" 
+                            Height="460"
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample9_xaml.txt:3]: I'm tempted to remove this as well. It's an app-specific implementation detail that probably shouldn't be reflected in the sample code. We don't want app developers in general hardcoding heights like this.

File: XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample9_xaml.txt [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+                            Header="This is Header Text" 
+                            PaneDisplayMode="$(PaneDisplay)" 
+                            IsTabStop="False"
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample9_xaml.txt:6]: This line also feels odd. NavigationView itself isn't a tab stop, is it? So this is redundant?

File: XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample9_xaml.txt [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    <muxc:NavigationView.MenuItems>
+        <muxc:NavigationViewItem Content="Calendar" Tag="SamplePage1" Icon="Calendar" />
+        <muxc:NavigationViewItem Content="Camera" Tag="SamplePage2" Icon="Camera" />
+        <muxc:NavigationViewItem Content="Notepad" Tag="SamplePage3" Icon="Edit" />
+    </muxc:NavigationView.MenuItems>
+    <muxc:NavigationView.FooterMenuItems>
+        <muxc:NavigationViewItem Content="Profile" Tag="SamplePage4" Icon="Contact" />
+        <muxc:NavigationViewItem Content="Delete" Tag="SamplePage5" Icon="Delete" />
+    </muxc:NavigationView.FooterMenuItems>
+    <Frame Margin="0,0,0,0" x:Name="contentFrame9" />
```
Comment (YuliKl) [inline XamlControlsGallery/ControlPagesSampleCode/NavigationView/NavigationViewSample9_xaml.txt:18]: That margin isn't doing anything, so shouldn't be in the sample code.

**Comment by anawishnoff** [comment]
Thanks @yulikl! I've made all of your suggested changes.

File: XamlControlsGallery/ControlPages/NavigationViewPage.xaml [hunk 7]
```diff
@@ -224,6 +224,56 @@
             </Grid>
         </local:ControlExample>
 
+        <local:ControlExample HeaderText="NavigationView with Footer Menu Items"
+                              VerticalAlignment="Top" 
+                              HorizontalContentAlignment="Stretch"
+                              XamlSource="NavigationView\NavigationViewSample9_xaml.txt"
+                              WebViewHeight="950">
```
Comment (stmoy) [inline XamlControlsGallery/ControlPages/NavigationViewPage.xaml:231]: Not sure why WebViewHeight is set here and in other places. Does this behave correctly even if this is removed? I see it's on other samples on this page too, but I suspect it doesn't actually do anything.

**Review by stmoy — APPROVED** [review]

File: XamlControlsGallery/ControlPages/NavigationViewPage.xaml (hunk 7 above)
Comment (anawishnoff) [reply XamlControlsGallery/ControlPages/NavigationViewPage.xaml]: Not sure either. I removed it and all is fine.

**Comment by anawishnoff** [comment]
@stmoy Not sure why these tests are failing. Any ideas?

**Comment by stmoy** [comment]
> @stmoy Not sure why these tests are failing. Any ideas?

The tests have been broken for awhile now and I haven't had the chance to fix/disable them. No worries that they are still failing - your change did not introduce the regressions.

**Review by YuliKl — APPROVED** [review]

**Pull request merged**